code/datums/components/stationloving.dm

```diff
@@ -50,6 +51,9 @@
 /datum/component/stationloving/proc/check_soul_imbue()
 	return disallow_soul_imbue
 
+/datum/component/stationloving/proc/check_mark_retrieval()
+	return TRUE
```

Comment:
Even if there's only one value make it a different define since in theory all signal returns are flags even if you're only using one of the bits. Check the other signals that return a flag for an example